components/cli/vendor/github.com/docker/docker/api/types/volume.go

```diff
@@ -44,15 +44,23 @@ type Volume struct {
 	UsageData *VolumeUsageData `json:"UsageData,omitempty"`
 }
 
-// VolumeUsageData volume usage data
+// VolumeUsageData Usage details about the volume. This information is used by the
```

Comment:
this change is a result of not having #132 also vendor the changes to the `components/engine` dir